src/htr2hpc/train.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+import argparse
+import os
+import sys
+
+import parsl
+from parsl.configs.local_threads import config
+
+
+from htr2hpc.train_apps import prep_training_data, segtrain
+
+
+api_token_env_var = "ESCRIPTORIUM_API_TOKEN"
+
+if __name__ == "__main__":
+    try:
+        api_token = os.environ[api_token_env_var]
+    except KeyError:
+        print(
+            f"Error: eScriptorium API token must be set as environment variable {api_token_env_var}",
+            file=sys.stderr,
+        )
+        sys.exit(1)
+
+    parser = argparse.ArgumentParser(
+        description="Export content from eScriptorium and train or fine-tune models"
+    )
+    parser.add_argument(
+        "base_url",
+        help="Base URL for eScriptorium instance (do not include /api/)",
+        type=str,
+    )
+    parser.add_argument(
+        "-d", "--document", help="Document id to export", type=int, dest="document_id"
+    )
+    parser.add_argument(
+        "-t", "--transcription", help="Transcription id to export", type=int
+    )
```

Comment:
A few suggested additions:
```
parser.add_argument(
	"-j", "--job", help="Job type (segmentation or transcription)", type=str
)
parser.add_argument(
	"-m", "--model", help="Optional model id to export (if fine-tuning)", type=int
)
parser.add_argument(
	"-p", "--parts", help="Optional list of parts to export (if not entire document)"
)
```

Reply:
@cmroughan these look like good options to add, do you want to use the suggest changes so it will show as your commit? (and/or you can send me your co-author commit email)

Reply:
Ah yes, I did forget about the add a suggestion option!